### PR TITLE
Capstone project Conference page

### PR DESCRIPTION
**Pull request by franclobo**
### Capstone project English Courses

1. Personalize the content of your page. 
2. Choose a different topic from the one in the original design.
3. Follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

- Colors.
- Typography: font face, size, and weight.
- Layout: composition and space between elements.

4. The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
5. Build only these 2 pages:
- The home page.
- The about page.

Each of these pages have versions for 2 different screen sizes:
- Mobile: up to 768px wide.
- Desktop: 768px or wider.

6. Interactions

- Links

The home page has a link to the about page in the menu.
The logo in the header links to the home page.

- Mobile menu

- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- **There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).**
- The mobile menu has a close (X) button that closes the menu.

7. Dynamic page

- The section "Featured speakers" has been created dynamically in JavaScript.
- Use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

8. Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      </div>
+      <div class="navigation">
+        <a><img class="mobile-menu" src="./icons/bars-solid.svg" alt="menu"></a>
+        <a href="./index.html"><img class="logo" src="./Pictures/logo.png" alt="logo"></a>
+        <div class="dropdown-menu hide-menu">
+          <figure class="menu-header">
+            <button class="close-menu">&times;</button>
+          </figure>
+          <ul class="mobile-menu-navigation">
+            <li><a>Home</a> </li>
```
Comment (Lordkaito) [inline index.html]: - [x] You are missing the link to the home page here and also on the about page, so I cannot go back to the home page once I got into the about page, so please work on this issue.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="./styles.css" rel="stylesheet">
+  <title>English courses</title>
+</head>
+<body class="container">
```
Comment (Lordkaito) [inline index.html:10]: - [x] If you check your page carefully, you will realize a white margin all around your page which is not supposed to be there, so please work on this issue to try to fix it.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @franclobo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @franclobo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**